grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -266,6 +271,130 @@ def testTotalRowCountIncludesAllChunks(self):
 
     self.assertEqual(progress.total_row_count, row_count)
 
+  def testFlowTruncatesFileAboveSingleLimit(self):
+    with temp.AutoTempFilePath() as temp_file_path:
+      target_bytes = 2**20  # 1 MiB
+
+      phrase = "This is a > 1 MiB file\n"
```

Comment:
I wonder if this could be written shorter:
(I didn't execute the code below)
```
with io.open(temp_file_path, "wb") as fd:
  fd.write(b"1" * (2**20 + 1))
```